django_vite/core/asset_loader.py

```diff
@@ -212,6 +222,8 @@ class DjangoViteAppClient:
     DjangoViteConfig provides the arguments for the client.
     """
 
+    ManifestClient = ManifestClient
```

Comment:
this seems a bit redundant?

Reply:
Defining the ManifestClient as a class variable allows you to create a subclassed DjangoViteAppClient with a separate subclassed ManifestClient. You can look at the `test_custom_app_client_class` for an example. If we didn't do this, then we'd have no way to overwrite the logic of the ManifestClient without doing module-level monkeypatching.

Thinking this through, we should probably also add other dependency classes like TagGenerator as class variables as well, to allow them to be easily extensible.